plugins/module_utils/acme/acme.py

```diff
@@ -82,6 +82,9 @@ def __init__(self, module, account):
             for key in ('newNonce', 'newAccount', 'newOrder'):
                 if key not in self.directory:
                     raise ModuleFailException("ACME directory does not seem to follow protocol ACME v2")
+            # Make sure that 'meta' is always available
+            if 'meta' not in self.directory:
+                self.directory = {}
```

Comment:
Thanks @ConsciaDBA! That was a stupid mistake :)
```suggestion
                self.directory['meta'] = {}
```